### DISCUSSION TITLE
Stream Quality 1080 or more?

### DISCUSSION THREAD

**Discussion by ayzick** [vzhd1701/gridplayer#57] [Q&A]
As the title says, Is there anyway to make The Stream Quality 1080p or more? cuz my Stream Quality only goes up to 720p
Can this be changed?
![Screenshot (9)](https://user-images.githubusercontent.com/68361570/176607739-f24186e2-0b4f-4cf3-a78c-1b4b912f0a34.png)

**Comment by vzhd1701** [comment]
For YouTube videos?

**Reply by ayzick** [reply]
Yes

**Reply by ayzick** [reply]
So How do I do it?

**Reply by vzhd1701** [reply]
You can't with the current version. There is a technical complication since YouTube 1080p streams come separate from the audio track and libVLC is bad at combining separate tracks together.

**Reply by ayzick** [reply]
oh

**Reply by ayzick** [reply]
but u planning to fix it?

**Reply by vzhd1701** [reply]
Yes, I am currently trying to find a way to make it work.

**Reply by ayzick** [reply]
ok gud

**Reply by ayzick** [reply]
Is it fixed?

**Reply by vzhd1701** [reply]
Not yet, sorry. To fix it I'll have to replace VLC library with MPV, which is going to take a while.

**Reply by ayzick** [reply]
Oh

**Reply by ayzick** [reply]
So I just saw the recent post on the Issues, So I could have just used yt-dlp instead of streamlink to be able to change the stream quality?

**Reply by vzhd1701** [reply]
GridPlayer always use yt-dlp for youtube videos. And it doesn't matter what plugin is used to extract stream links, the problem is that libVLC that GridPlayer uses to play videos can't handle joining separate video and audio streams on the fly. Please be patient, I'll fix it in the future.

**Reply by ayzick** [reply]
Aight den

**Reply by ayzick** [reply]
Is this Update coming out Next year?

**Reply by ayzick** [reply]
I am still waiting
